docs/src/main/asciidoc/kafka.adoc

```diff
@@ -2401,6 +2401,95 @@ quarkus.log.category."org.apache.kafka.common.utils".level=INFO
+----
+kafka.bootstrap.servers=my-event-hub.servicebus.windows.net:9093 # <1>
+kafka.security.protocol=SASL_SSL
+kafka.sasl.mechanism=PLAIN
+kafka.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required \ # <2>
+    username="$ConnectionString" \ # <3>
+    password="<YOUR.EVENTHUBS.CONNECTION.STRING>"; # <4>
+----
+<1> The port is `9093`.
+<2> You need to use the JAAS `PlainLoginModule`.
+<3> The username is the `$ConnectionString` string.
+<4> The Event Hub connection string given by Azure.
+
+Replace `<YOUR.EVENTHUBS.CONNECTION.STRING>` with the connection string for your Event Hubs namespace.
+For instructions on getting the connection string, see https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string[Get an Event Hubs connection string].
+The result would be something like:
+
+[source, properties]
+----
+kafka.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required \
+    username="$ConnectionString" \
+    password="Endpoint=sb://my-event-hub.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=XXXXXXXXXXXXXXXX";
+----
+
+This configuration can be global (as above), or set in the channel configuration:
+
+[source, properties]
+----
+mp.messaging.incoming.$channel.bootstrap.servers=my-event-hub.servicebus.windows.net:9093
+mp.messaging.incoming.$channel.security.protocol=SASL_SSL
+mp.messaging.incoming.$channel.sasl.mechanism=PLAIN
+mp.messaging.incoming.$channel.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required \
+    username="$ConnectionString" \
+    password="Endpoint=sb://my-event-hub.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=...";
+----
+
+=== Red Hat OpenShift Streams for Apache Kafka
+
+https://cloud.redhat.com/[Red Hat OpenShift Streams for Apache Kafka] provides managed Kafka instances.
+Once you have created your Kafka instance, and copied the client id/client secret, you can configure the Quarkus application to connect to the broker as follows:
+
+[source, properties]
+----
+kafka.bootstrap.servers=<connection url> # <1>
+kafka.security.protocol=SASL_SSL
+kafka.sasl.mechanism=PLAIN
+kafka.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required \
+  username="${KAFKA_USERNAME}" \ # <2>
+  password="${KAFKA_PASSWORD}"; # <3>
+----
+<1> The connection string, given on the admin console, such as `demo-c--bjsv-ldd-cvavkc-a.bf2.kafka.rhcloud.com:443`
+<2> The kafka username (the client id from the service account)
+<3> the kafka password (the client secret from the service account)
+
+NOTE: In general, these properties are prefixed using `%prod` to enable them only when running in production mode.
+
+IMPORTANT: To use Red Hat OpenShift Streams for Apache Kafka, you must create the topic beforehand, create a _Service Account_, and provide permissions to read and write to your topic from that service account.
```

Comment:
Would be useful to add a link to this guide for these operations:
 https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/88e1487a-2a14-4b35-85b9-a7a2d67a37f3

Reply:
How stable is this link? My bookmark are regularly out of date.

Reply:
Yeah, I agree. I have some 404 pages through the web archive.

Reply:
@cescoffier I have a blog post that specifically shows how to do this using the Quarkus Superheroes sample app - https://developers.redhat.com/articles/2022/03/23/quarkus-superheroes-managed-services-save-day

Feel free to link to it if you feel its appropriate.

Reply:
ah yes, good idea!

Reply:
After verification, the https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/88e1487a-2a14-4b35-85b9-a7a2d67a37f3 should be stable (that's what took me some time to come back to this PR).